Documentation/getting-started.md

```diff
@@ -0,0 +1,78 @@
+# Getting Started with Analytics SDK
+
+## Before starting
+
+Analytics extension has a dependency on [AEP Core SDK](https://github.com/adobe/aepsdk-core-android#readme) which must be installed to use the extension.
+
+## Add Analytics extension to your app
+
+1. Installation via [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/) is the easiest and recommended way to get the AEP SDK into your Android app. Add a dependency on Analytics and Core to your mobile application. To ensure consistent builds, it is best to explicitly specify the dependency version and update them manually.
+
+   ```
+   implementation 'com.adobe.marketing.mobile:core:2.+'
+   implementation 'com.adobe.marketing.mobile:analytics:2.+'
+   ```
```

Comment:
Add a like to our latest release page, similar to the example for Edge, https://github.com/adobe/aepsdk-edge-android/blob/dev-v2.0.0/Documentation/getting-started.md#add-the-edge-network-extension-to-your-app